tests/testthat/_snaps/bin2factor.md

```diff
@@ -14,15 +14,17 @@
       rec %>% step_bin2factor(rocks, levels = letters[1:5])
     Condition
       Error in `step_bin2factor()`:
-      ! `levels` should be a two element character string
+      x `levels` should be a two element character string.
```

Comment:
```suggestion
      x `levels` should be a 2-element character string.
```

Just so that we don't mix numbers and spellings of numbers.

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350